nodes/src/nodes/tool_github/IInstance.py

```
@@ -0,0 +1,1046 @@
+    def file_get(self, args):
+        args = _normalize(args)
+        repo = self._repo(args)
+        path = _require_str(args, 'path', 'file_get')
+        params = {'ref': args['ref']} if args.get('ref') else None
+        data = call(self._token(), 'GET', f'/repos/{repo}/contents/{path.lstrip("/")}', params=params)
+        if isinstance(data, list):
+            raise ValueError(f'Path "{path}" is a directory — use file_list instead')
+        content_b64 = data.get('content', '')
+        try:
+            content = base64.b64decode(content_b64).decode('utf-8', errors='replace')
+        except Exception:
```

Ruff (0.15.9)

[warning] Do not catch blind exception: `Exception`

(BLE001)